### PR TITLE
add globalhub inspect

### PR DESCRIPTION
**Pull request by hchenxa**
**Related Issue:**  stolostron/backlog#<ISSUE_NUMBER>
 https://issues.redhat.com/browse/ACM-9153

**Description of Changes:**
 the globalhub namespace in incorrect

**What resource is being added**: <resource name> | N/A

**Is this a Hub or Managed cluster change?:**
Hub Cluster | Managed Cluster | N/A

**Notes:**

### CODE REVIEW

File: collection-scripts/gather [hunk 1]
```diff
@@ -256,6 +259,31 @@ gather_hub(){
...
+      oc adm inspect kafkas.kafka.strimzi.io --all-namespaces --dest-dir=must-gather
+      oc adm inspect kafkatopics.kafka.strimzi.io --all-namespaces --dest-dir=must-gather
+      oc adm inspect kafkausers.kafka.strimzi.io --all-namespaces --dest-dir=must-gather
+      KAFKANS=`oc get kafkas.kafka.strimzi.io --all-namespaces --no-headers=true -o custom-columns=NAMESPACE:.metadata.namespace | sort -u`
+      for kns in $KAFKANS;
+      do
+          oc adm inspect ns/"$kns"  --dest-dir=must-gather
+      done
+      # postgresql
+      oc adm inspect postgresclusters.postgres-operator.crunchydata.com --all-namespaces --dest-dir=must-gather
```
Comment (clyang82) [inline collection-scripts/gather:276]: Need to modify here or just remove it. we do not create postgrescluster anymore.

File: collection-scripts/gather [hunk 2]
```diff
@@ -256,6 +259,31 @@ gather_hub(){
     oc adm inspect submarinerconfigs.submarineraddon.open-cluster-management.io --all-namespaces --dest-dir=must-gather
     oc adm inspect brokers.submariner.io --all-namespaces --dest-dir=must-gather
 
+    # Multicluster GlobalHub information
+    oc adm inspect multiclusterglobalhubs.operator.open-cluster-management.io --all-namespaces --dest-dir=must-gather
+    if [ $? = 0 ];
+    then
+      # kafka
+      oc adm inspect kafkas.kafka.strimzi.io --all-namespaces --dest-dir=must-gather
```
Comment (clyang82) [inline collection-scripts/gather:267]: maybe target to global hub namespace only?

**Comment by clyang82** [comment]
Do you want to keep consistent with main branch? or you want to address my comments?

**Comment by hchenxa** [comment]
yes, keep the same behavior with main branch

**Pull request merged**